Source/Core/DolphinWX/MainMenuBar.cpp

```diff
@@ -216,6 +216,10 @@ wxMenu* MainMenuBar::CreateToolsMenu() const
   tools_menu->Append(IDM_IMPORT_SAVE, _("Import Wii Save..."));
   tools_menu->Append(IDM_EXPORT_ALL_SAVE, _("Export All Wii Saves"));
   tools_menu->AppendSeparator();
+  tools_menu->Append(IDM_LOAD_GC_BIOS_EUR, _("Load GameCube BIOS (EUR)"));
+  tools_menu->Append(IDM_LOAD_GC_BIOS_JAP, _("Load GameCube BIOS (JAP)"));
+  tools_menu->Append(IDM_LOAD_GC_BIOS_USA, _("Load GameCube BIOS (USA)"));
```

Comment:
Since these are user-visible strings, please use more proper names for the regions (for instance, instead of USA, either NTSC-U (which is what I prefer) or something like North America)